api/io/no-features.txt

```diff
@@ -141,7 +141,7 @@ pub mod bitcoin_io
 pub struct bitcoin_io::Cursor<T>
 pub struct bitcoin_io::Error
 pub struct bitcoin_io::Sink
-pub struct bitcoin_io::Take<'a, R: bitcoin_io::Read + ?core::marker::Sized>
+pub struct bitcoin_io::Take<'a, R: bitcoin_io::Read>
```

Comment:
Whaaat? You reintroduced these... :(

Reply:
You weren't around to write the tooling required to run `cargo public-api` in CI so we could use the output without committing the files. And I couldn't be bothered to ...